db/schema.rb

```diff
@@ -169,10 +169,11 @@
     t.text "consumer_key", null: false
     t.text "shared_secret", null: false
     t.text "lms_link"
+    t.bigint "organization_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.bigint "organization_id"
```

Comment:
Lol why did this column want to move so badly 😂

Reply:
I don't know 😭 This also happened with @thebradbain's PR too.